CHANGES.rst

```diff
@@ -18,6 +18,11 @@ Major changes
 Minor changes
 -------------
 
+* The :func:`patch_display` function has been added. It changes the display of
```

Comment:
I think that this entry should not be under "minor change". Maybe a section "new features"?